napari/components/layerlist.py

```diff
@@ -62,6 +62,26 @@ def selected(self):
         """List of selected layers."""
         return [layer for layer in self if layer.selected]
 
+    @property
+    def active(self):
+        """Active layer, if any, given by top most selected."""
+        active_layer = None
+        # if multiple layers are selected sets the active layer to None
+        for layer in self:
+            if active_layer is None and layer.selected:
+                active_layer = layer
+            elif active_layer is not None and layer.selected:
+                active_layer = None
+                break
+        return active_layer
+
+    @active.setter
+    def active(self, active):
+        if active not in self:
+            raise ValueError('Active layer must be in layer list')
+        self.unselect_all(ignore=active)
```

Comment:
Double iteration over hole layer list which may cause slowness when adding multiple layers.

Reply:
I don't think we ever call this setter internally, it's only there for user external user convenience